nautobot_netbox_importer/tests/fixtures/netbox_dump.json

```diff
@@ -3729,6 +3729,35 @@
             "comments": ""
         }
     },
+    {
+        "model": "circuits.providernetwork",
+        "pk": 1,
+        "fields": {
+            "created": "2021-02-21",
+            "last_updated": "2021-02-21T23:53:40.570Z",
+            "custom_field_data": {},
+            "name": "ProviderNetwork A",
+            "slug": "providernetwork-a",
+            "provider": 1,
+            "comments": "",
+            "description": ""
+        }
+    },
+    {
+        "model": "circuits.providernetwork",
+        "pk": 2,
+        "fields": {
+            "created": "2021-02-21",
+            "last_updated": "2021-02-21T23:53:40.570Z",
+            "custom_field_data": {"b": "b"},
+            "name": "ProviderNetwork B",
+            "slug": "providernetwork-b",
+            "provider": 2,
+            "service_id": "pnb",
```

Comment:
so, the `service_id` field is lost in translation for now.
I would assume that this works as it is (i.e. it is ignored), but next step is to do something such as raise warning or create a custom_field (e.g. `netbox_service_id`) ?

Reply:
good, I like the idea to get a consistent approach for all similar cases

Reply:
https://github.com/nautobot/nautobot-plugin-netbox-importer/pull/101